src/wasm-lib/kcl/src/std/sketch.rs

```diff
@@ -1043,10 +1034,11 @@ async fn inner_start_sketch_on(
     args: &Args,
 ) -> Result<SketchSurface, KclError> {
     match data {
-        SketchData::Plane(plane_data) => {
-            let plane = start_sketch_on_plane(plane_data, exec_state, args).await?;
+        SketchData::PlaneOrientation(plane_data) => {
+            let plane = make_sketch_plane_from_orientation(plane_data, exec_state, args).await?;
             Ok(SketchSurface::Plane(plane))
         }
+        SketchData::Plane(plane) => Ok(SketchSurface::Plane(plane)),
```

Comment:
This is never being triggered, even in KCL like this:

```kcl
offsetPlane001 = offsetPlane("XY", 10)
sketch001 = startSketchOn(offsetPlane001)
  |> circle({ center: [0.05, 0.14], radius: 6.45 }, %)
```

Even though [I altered the `offset_plane` function to return a full `Plane`](https://github.com/KittyCAD/modeling-app/pull/4481/files#diff-bfd17ddbae4dbd6850473f927f670dce0eb31e7acd478300fe7c3263d39eb305L166-R163) instead of what is now called `PlaneOrientationData`. Why is `sketch001 = startSketchOn(offsetPlane001)` acting like it has `PlaneOrientation` instead of a `Plane`? Maybe I'm _not_ actually passing through a full reference to a `Plane` with `offset_plane`?

Reply:
This was happening because `#[serde(untagged)]` is present, and a sub-variant of the first enum variant `PlaneOrientationData::Custom(...)` was a struct that is a subset of `Plane`, so it would always match first from serde's point of view. Undoing my newtype shenanigans made this hard thinking unnecessary.